### ISSUE TITLE
Add PlaceholderAPI support.

### ISSUE THREAD

**Issue by seyfahni** [seyfahni/OnlineTime#9]
Add [PlaceholderAPI](https://www.spigotmc.org/resources/placeholderapi.6245/) support:
```
| Placeholder               | Replacement (example time: 5d 2h 7min) |
| ------------------------- | -------------------------------------- |
| %online_time%             | 5 days 2 hours 7 minutes               |
| %online_time_seconds%     | 0                                      |
| %online_time_minutes%     | 7                                      |
| %online_time_hours%       | 2                                      |
| ...                       | ...                                    |
| %online_time_all_seconds% | 439620                                 |
| %online_time_all_minutes% | 7327                                   |
| %online_time_all_hours%   | 122                                    |
| %online_time_all_days%    | 5                                      |
| %online_time_all_weeks%   | 0                                      |
| ...                       | ...                                    |
```

**Comment by seyfahni** [comment]
Implemented in b249ab04f9fcfb4d0bc41015a873f0b8d43b1084.

**Comment by seyfahni** [comment]
Actual implementation works as follows:
```
| Placeholder                 | Description                                       | Example (with 5d 2h 7min)   |
| --------------------------- | ------------------------------------------------- | --------------------------- |
| `%onlinetime_string%`       | formatted string (same as when using /onlinetime) | `5 days 2 hours 7 minutes`  |
| `%onlinetime_only_seconds%` | seconds, as in formatted string (without unit)    | `0`                         |
| `%onlinetime_only_minutes%` | minutes, as in formatted string (without unit)    | `7`                         |
| `%onlinetime_only_hours%`   | hours, as in formatted string (without unit)      | `2`                         |
| `%onlinetime_only_days%`    | days, as in formatted string (without unit)       | `5`                         |
| `%onlinetime_only_weeks%`   | weeks, as in formatted string (without unit)      | `0`                         |
| `%onlinetime_only_months%`  | months, as in formatted string (without unit)     | `0`                         |
| `%onlinetime_only_years%`   | years, as in formatted string (without unit)      | `0`                         |
| `%onlinetime_all_seconds%`  | whole time in seconds                             | `439620`                    |
| `%onlinetime_all_minutes%`  | whole time in minutes                             | `7327`                      |
| `%onlinetime_all_hours%`    | whole time in hours                               | `122`                       |
| `%onlinetime_all_days%`     | whole time in days                                | `5`                         |
| `%onlinetime_all_weeks%`    | whole time in weeks                               | `0`                         |
| `%onlinetime_all_months%`   | whole time in months                              | `0`                         |
| `%onlinetime_all_years%`    | whole time in years                               | `0`                         |
```
